gulpfile.js

```diff
@@ -144,23 +159,31 @@ gulp.task('watch', [ 'jstest-watch', 'webpack-watch' ], () => {
   gulp.watch(paths.SASS, ['sasstest']);
 
   // run webpack to compile svg icons and styleguide assets
-  gulp.watch([
-    `${paths.ICONS_DIR}**/*.svg`,
-    `${paths.SRC_TEMPLATES_DIR}_icon_template.lodash`,
-    './STYLEGUIDE.md'
-  ], ['webpack']);
+  gulp.watch(
+    [
+      `${paths.ICONS_DIR}**/*.svg`,
+      `${paths.SRC_TEMPLATES_DIR}_icon_template.lodash`,
+      './STYLEGUIDE.md'
```

Comment:
JS doesn't deal well with trailing commas, does it? That'd result in `[foo, bar, undefined]` or whatever, right? Pity if so, because they do make diffs nicer.